packages/keystone/KSv5v6/v5/keystone.js

```diff
@@ -0,0 +1,15 @@
+const { Keystone: DefaultKeystone } = require('@keystonejs/keystone')
+
+const { _patchResolvers } = require('../utils/resolvers')
+
+class Keystone extends DefaultKeystone {
+    getResolvers ({ schemaName }) {
+        const originalResolvers = super.getResolvers({ schemaName })
+
+        return _patchResolvers(originalResolvers)
```

Comment:
`_patchResolvers` is too general a name for this function. Maybe specify what really you patch here?

Reply:
I'll think about naming here